src/classes/Ingredient.js

```diff
@@ -18,7 +18,7 @@ class Ingredient {
     allIngredientData.name = fromIngredients.name;
     allIngredientData.cost = fromIngredients.estimatedCostInCents;
     allIngredientData.amount =
-      ingredientSource.amount || ingredientSource.quantity.amount;
+      ingredientSource.amount || ingredientSource.quantity?.amount;
     if (ingredientSource.quantity) {
       allIngredientData.unit = ingredientSource.quantity.unit;
```

Comment:
The `optional chaining` works great here! We no longer are getting a null for the amount when we load the user.